scss/style.scss

```
@@ -0,0 +1,626 @@
+    margin-bottom: 95px;
+    display: flex;
+    justify-content: flex-start;
+    align-items: flex-end;
+
+    &__text-block {
+      width: 428px;
+      margin-left: 168.5px;
+    }
+
+    h2 {
+      color: #0b132a;
+      font-size: 35px;
+      font-weight: 500;
+      line-height: 50px; /* 142.857% */
+      filter: blur(1px) brightness(50%);
+    }
+
+    &__text-block p {
+      color: #4f5665;
+      font-size: 16px;
+      line-height: 30px; /* 187.5% */
+    }
+
+    &__text-block ul {
+      padding: 0;
+    }
+
+    &__text-block ul li {
+      color: #4f5665;
+      font-size: 14px;
+      line-height: 30px; /* 214.286% */
+      padding: 3px 0;
+      list-style: none;
+    }
+
+    &__text-block ul li::before {
+      content: url(../images/main/section_2/check.svg);
+      position: relative;
+      top: 7px;
+      right: 4px;
+    }
+  }
+
+  .section-third {
+    width: 1440px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    background: linear-gradient(
+      180deg,
+      #f8f8f8 -45.04%,
+      rgba(248, 248, 248, 0) 88.56%
+    );
+
+    h3 {
+      text-align: center;
+      font-size: 35px;
+      font-weight: 500;
+      line-height: 50px; /* 142.857% */
+      margin-top: 80px;
+      margin-bottom: 20px;
+    }
+
+    & p {
+      color: #4f5665;
+      text-align: center;
+      font-size: 16px;
+      line-height: 30px; /* 187.5% */
+      width: 555px;
+      margin: 0;
+    }
+
+    &__tariff {
+      display: flex;
+      justify-content: center;
+    }
+
+    &__plans {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      justify-content: space-between;
+      width: 330px;
+      height: 760px;
+      border-radius: 10px;
+      border: 2px solid #ddd;
+      background: #fff;
+      margin: 60px 25px;
+      padding-top: 79px;
+      padding-bottom: 50px;
+      box-shadow: 5px 5px 20px #4f5665;
+    }
+
+    &__plans:nth-child(3) {
+      border: 2px solid #f53838;
+    }
+
+    &__plans-block {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      justify-content: center;
+    }
+
+    &__plans p {
+      color: #0b132a;
+      text-align: center;
+      font-size: 18px;
+      font-weight: 500;
+      line-height: 30px; /* 166.667% */
+      margin-top: 30px;
+    }
+
+    &__price {
+      font-size: 25px;
+    }
+
+    &__price span {
+      color: #4f5665;
+      font-size: 25px;
+      line-height: 30px;
+    }
+
+    &__plans ul {
+      padding-left: 20px;
+      margin-top: 30px;
+      margin-bottom: 20px;
+    }
+
+    &__plans ul li {
+      color: #4f5665;
+      font-size: 14px;
+      line-height: 30px; /* 214.286% */
+      list-style-type: none;
+      margin-bottom: 10px;
+    }
+
+    &__plans ul li::before {
+      content: url(../images/main/section_3/check.svg);
+      position: relative;
+      top: 7px;
+      right: 15px;
+    }
```

ми ж як раз можемо все сходинками прописати
```suggestion
    &__plans ul {
      padding-left: 20px;
      margin-top: 30px;
      margin-bottom: 20px;
      
      li {
        color: #4f5665;
        font-size: 14px;
        line-height: 30px; /* 214.286% */
        list-style-type: none;
        margin-bottom: 10px;
        
        &::before {
          content: url(../images/main/section_3/check.svg);
          position: relative;
          top: 7px;
          right: 15px;
        }
      }
    }
```